lib/index.js

```diff
@@ -27,12 +28,12 @@ if (options.version) {
 }
 
 var updateIndex = function(data, callback) {
-    var index = path.join(options.dir, 'index.json');
-    fs.readFile(index, 'utf8', function(err, d) {
-        if (d) {
+    hooks.globalIndexJson(data, callback, function() {
```

Comment:
Same as the `tarball` hook, missing the arguments and the action to stop the write.